open-api/src/testFixtures/java/org/apache/iceberg/rest/RCKUtils.java

```diff
@@ -76,15 +78,22 @@ static Map<String, String> environmentCatalogConfig() {
                 HashMap::new));
   }
 
-  static RESTCatalog initCatalogClient() {
+  public static RESTCatalog initCatalogClient() {
```

Comment:
I think we can get away with leaving this as package protected and not expose the inner workings of the RCK and Extension.  Take a look at my example below.

Reply:
Done